plugin/supervisor.go

```diff
@@ -111,13 +108,8 @@ func (sup *supervisor) Hooks() Hooks {
 	return sup.hooks
 }
 
-// PerformHealthCheck checks the plugin through a process check, an RPC ping, and a HealthCheck hook call.
+// PerformHealthCheck checks the plugin through an an RPC ping.
```

Comment:
Did we ever have a `HealthCheck` call?

Reply:
@lieut-data Yes, it was originally part of the PR that introduced the health check, but there was not a consensus on how it should work, so it was removed. See https://github.com/mattermost/mattermost-server/pull/10462#discussion_r279267206. My apologies for leaving this comment this way.

Reply:
No worries! Just wanted to make sure I didn't miss anything :D